lib/commands/command-plugin-update.bash

```diff
@@ -10,11 +10,12 @@ plugin_update_command() {
   local gitref="${2:-master}"
   if [ "$plugin_name" = "--all" ]; then
     if [ -d "$(asdf_data_dir)"/plugins ]; then
+      plugins=(find "$(asdf_data_dir)"/plugins -type d -mindepth 1 -maxdepth 1)
       while IFS= read -r -d '' dir; do
         local plugin_name
         plugin_name=$(basename "$dir")
         update_plugin "$plugin_name" "$dir" "$gitref" &
-      done < <(find "$(asdf_data_dir)"/plugins -type d -mindepth 1 -maxdepth 1)
+      done <<< "$plugins"
```

Comment:
This `find` was introduced because the `for dir in plugins/*` didn't handle the case where the `plugins/` dir was not yet created. Is `find` more portable than `for..in` and would it be worth us either replacing all similar usages to the more portable version? (and in this specific case just creating the empty dir to address the root issue)

Reply:
I think `find` is more portable in general. There my be newer `find` flags that aren't widely available, but the existing flags are sufficient for our uses cases so I don't think it would matter.